lib/screens/guest-details.dart

```diff
@@ -340,108 +318,163 @@ SliverToBoxAdapter mustAssignTicketsText () {
-                                horizontal: 16, vertical: 5),
-                            child: Container(
-                                width: 300,
-                                height: 45,
-                                child: TypeAheadField(
-                                  hideOnEmpty: true,
-                                  textFieldConfiguration: TextFieldConfiguration<Guest>(
-//                                                autofocus: this.index == 1,
-                                      controller: textFieldController,
-                                      onChanged: (dynamic value) {
-                                        record.setName(value.toString());
-                                      },
-                                      onSubmitted: (dynamic value) {
-                                        _validateGuestTickets(context, owner);
-                                      },
-                                      onEditingComplete: () {
-                                        _validateGuestTickets(context, owner);
-                                      },
-                                      style: appTheme.textTheme.headline1,
-                                      textCapitalization: TextCapitalization.words,
-                                      decoration: InputDecoration(
-                                        border: OutlineInputBorder(),
-                                        labelText: ticketLabel,
-                                      )
-                                  ),
-                                  suggestionsCallback: (search) async {
-                                    return _filterGuestsByName(context, search);
-                                  },
-                                  itemBuilder: (context,
-                                      Guest guest) {
-                                    return ListTile(
-                                        title: Text(guest.name,
-                                            style: appTheme.textTheme
-                                                .headline3)
-                                    );
-                                  },
-                                  onSuggestionSelected: (Guest guest) {
-                                      record.setName(guest.name);
-                                      textFieldController.text = guest.name;
-                                      if (owner.contract.valid()) {
-                                        _validateGuestTickets(context, owner);
-                                      }
-                                  },
-                                )
-                            )
-                        ),
-                      ]
-                  )
-                ]
-            );
-          }
-      );
+                children: <Widget>[  
+                  Container(
+                    width: 150,
+                    child: Text(owner.name)
```

Comment:
Ideally, this should be the original purchaser's name instead of the owner's name